README.md

```diff
@@ -184,7 +196,13 @@ docker-compose build tooling-invoker
 
 You can then run `./bin/start` as normal.
 
-Each time you change the Dockerfile or dependencies it mounts (e.g. the Gemfile), you will need to rerun the build and start commands.
+Alternatively, you can skip the `docker-compose build` step and pass the `--build` flag to `./bin/start`:
+
+```bash
+./bin/start --build
+```
```

Comment:
As above, should we remove the `docker-compose --build ...` step as a seperate suggestion and just have this?